pkg/apis/core/types.go

```diff
@@ -3541,6 +3568,24 @@ const (
 	FSGroupChangeAlways PodFSGroupChangePolicy = "Always"
 )
 
+// SupplementalGroupsPolicy defines how supplemental groups
+// of the first container processes are calculated.
+type SupplementalGroupsPolicy string
+
+const (
+	// Merge policy always merges the provided SupplementalGroups (including FsGroup)
+	// specified in SecurityContext with groups of the primary user from the container
+	// image(`/etc/group`).
+	// Note: The primary user is specified with RunAsUser.
+	//       If not specified, the user from the image config is used.
+	//       Otherwise, the runtime default is used.
+	SupplementalGroupsPolicyMerge SupplementalGroupsPolicy = "Merge"
+	// Strict policy uses only the provided SupplementalGroups(including FsGroup)
```

Comment:
Wording

```
	// SupplementalGroupsPolicyStrict means that the container's provided
	// SupplementalGroups and FsGroup (specified in SecurityContext) will be
	// used instead of any groups defined in the container image.
```